index.html

```diff
@@ -0,0 +1,288 @@
+                        </nav>
+                        <nav class="right">
+                            <img src="Cellular.png" alt="Cellular">
+                            <img src="Wifi.png" alt="Wifi">
+                            <img src="Battery.png" alt="battery">
+                        </nav>
+                        <nav>
+                            <div class="toolbar1">
+                                <a href="#">My Logo</a>
+                            </div>
+                            <section class="header-images">
+                                <div id="elipse11">
+                                    <img src="Ellipse11.png" alt="">
+                                </div>
+                                <div id="elipse5">
+                                    <img src="Ellipse5.png" alt="">
+                                </div>
+                                <div id="rectangle7">
+                                    <img src="Rectangle7.png" alt="">
+                                </div>
+                                <div id="elipse3">
+                                    <img src="Ellipse3.png" alt="">
+                                </div>
+                                <div id="rectangle4">
+                                    <img src="Rectangle4.png" alt="">
+                                </div>
+                                <div id="elipse8">
+                                    <img src="Ellipse8.png" alt="">
+                                </div>
+                            </section>
+                            <div class="toolbar2">
+                                <img src="Union.png" alt="Menu">
+                            </div>
+                        </nav> 
+                    
+                    </div>
+                </header>
+                <section id="primary">
+                    <div class="name">
+                        <h1>I'm Kelvin Glad to see you!</h1>
+                    </div>
+                    <div class="text">
+                        <p>I'm a software developer! I can help you build a product,
+                            feature or website. Look through someof my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <nav id="lets-connect">
+                        <h5>LET'S CONNECT</h5>
+                    </nav>
+                    <div>
+                        <ul class="icons">
+                            <li><a href=""></a><img src="Vector.png" alt=""></li>
+                            <li><a href=""><img src="Linkedin.png" alt=""></a></li>
+                            <li><a href=""><img src="Vector1.png" alt=""></a></li>
+                            <li><a href=""><img src="Vector3.png" alt=""></a></li>
+                        </ul>
```

Comment:
- [ ] As suggested by the previous reviews, it would be better to use the correct image path, make sure you have those images in your directory and they are displayed on the screen. I don't have any images in your portfolio directory, maybe you forgot to download them from the Figma template. Kindly download them and ensure that the issue is fixed.